src/app/editor/form-generator/components/binary-field/binary-field.scss

```diff
@@ -45,6 +45,7 @@ $preview-max-width: 54%;
 }
 
 .image-preview {
+    position: relative;
     text-align: center;
     gentics-ui-image-preview {
         display: flex;
```

Comment:
All the style defs below this one (from `.video-preview` onwards) are no longer relevant and should be removed.

Additionally, the `.default-preview` rules should be transferred over to the new FilePreviewComponent styles.